src/components/AddForm/AddForm.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const AddForm = () => (
```

Comment:
- [ Optional] In a big application, you might have many forms to add things. Therefore, it would be good to use more descriptive names. For example, `AddBookForm`, etc